src/components/SubWheel.js

```diff
@@ -0,0 +1,29 @@
+const SubWheel = (props) => {
+    const buildLines = (subs) => {
```

Comment:
this is a component in disguise, if you use an object is better
```suggestion
    const Lines = ({subs}) => {
```